consoles/sshVirtsh.pm

```diff
@@ -505,17 +506,20 @@ sub add_disk {
     return;
 }
 
+sub virsh {
+    my $libvirt_connector = get_var('VMWARE_REMOTE_VMM', ' ');
+    return "virsh$libvirt_connector";
```

Comment:
`VMWARE_REMOTE_VMM`, if defined, may not start with space:
```suggestion
    return "virsh $libvirt_connector";
```

Reply:
true, updated. I hope I got it right.